src/v2/Apps/Show/ShowApp.tsx

```diff
@@ -57,11 +62,44 @@ export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
 // Top-level route needs to be exported for bundle splitting in the router
 export default createFragmentContainer(ShowApp, {
   show: graphql`
-    fragment ShowApp_show on Show {
+    fragment ShowApp_show on Show
+      @argumentDefinitions(
```

Comment:
This is the annoying repetition. The `ShowApp` fragment and container need to declare these just to 'pass them on' to the child container that expects these arguments. In GraphQL land (and if you look at the generated queries), arguments are global and so you don't need this (`@arguments` and `@argumentDefinitions` are Relay-specific directives).